doc/C_STYLE_GUIDE.rst.txt

```diff
@@ -129,16 +129,16 @@ Code lay-out
         }
 
 
-* The return statement should *not* get redundant parentheses::
+* The return statement should *not* get redundant parenthesis::
```

Comment:
"parentheses" is correct, "parenthesis" refers to the expression within the "parentheses"